radis/lbl/base.py

```diff
@@ -3267,9 +3267,9 @@ def _reinitialize(self):
                     "PerformanceWarning",
                 )
             else:
-                limit = (
-                    mem / 25 * 4
-                )  # the difference between deep=True and deep=False is around 4 times
+                limit = mem / 2  # 50 % of user RAM
+
+            # Note: the difference between deep=True and deep=False is around 4 times
```

Comment:
you didn't fix in the "object" case neither?

Reply:
Actually, this case might be deceiving a little. When using CH4 for 4 million lines, it was occupying 75% of RAM already and value of sf.df.memory_usage(deep=False) was only 4% of the total memory, that's why I doubted a larger threshold might lead to full usage of memory already.

Reply:
Ok. Let's try like this ; we'll see after usage and adjust if needed.